src/main.s

```diff
@@ -0,0 +1,72 @@
+.INCLUDE "../GBDefs/defs.s"
+.INCLUDE "hardware_specs.s"
+
+; ///////// Mapping \\\\\\\\\
+.INCLUDE "global.var.s"
+
+.ENUM $C000
+	global_ INSTANCEOF global_var
+.ENDE
+; \\\\\\\\\ Mapping /////////
+
+; ///////// DISABLE INTERRUPTIONS \\\\\\\\\
+.ORG $0040 				; Write at the address $0040 (vblank interuption)
+	call VBlank
+	reti
+
+.ORG $0100 				; Write at the address $0100 (starting point of the prog)
+	nop							; adviced from nintendo. nop just skip the line.
+	jp start
+; \\\\\\\\\ DISABLE INTERRUPTIONS /////////
```

Comment:
You're not disabling the interruptions in this sections, but actually handling VBlank interruption, and then handling the program start point.

Reply:
-_-, yep, I'm dumb